doc/examples/filters/plot_denoise.py

```diff
@@ -38,7 +38,7 @@
 noisy = astro + 0.6 * astro.std() * np.random.random(astro.shape)
```

Comment:
I'm wondering if using http://scikit-image.org/docs/dev/api/skimage.util.html#random-noise wouldn't be better... @jni just fixed this function, what's your opinion @jni?